lntest/itest/lnd_test.go

```diff
@@ -9031,30 +9031,41 @@ out:
 	if err := net.ConnectNodes(ctxt, net.Bob, carol); err != nil {
 		t.Fatalf("unable to connect bob to carol: %v", err)
 	}
-	ctxt, _ = context.WithTimeout(ctxb, channelOpenTimeout)
-	chanPoint = openChannelAndAssert(
-		ctxt, t, net, net.Bob, carol,
-		lntest.OpenChannelParams{
-			Amt: chanAmt,
-		},
-	)
 
 	// Reconnect Alice and Bob. This should result in the nodes syncing up
 	// their respective graph state, with the new addition being the
 	// existence of Carol in the graph, and also the channel between Bob
 	// and Carol. Note that we will also receive a node announcement from
 	// Bob, since a node will update its node announcement after a new
-	// channel is opened.
+	// channel is opened. We connect Alice and Bob here to ensure her
+	// gossip filter isn't set after the announcements between Bob and Carol
```

Comment:
This kinda changes what this test is checking. Previously it checked that a new node's announcement was synced on connection, now it connects first then the announcement is created.